extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -436,6 +437,9 @@ export class SchemaCompareOptionsDialog {
 	protected async execute() {
 		this.SetDeploymentOptions();
 		this.SetObjectTypeOptions();
+		if (this.optionsChanged) {
+			vscode.window.showInformationMessage(localize('schemaCompareOptions.recompareMessage', 'Options have changed. Press Compare to see the comparison.'));
```

Comment:
@angdesign should we pop up this one too?
we can pop this up if needed using MessageOptions: {modal:true} -

Reply:
hmm, can we go one step further here? ask whether you want to refresh the comparison, Yes or No, if Yes, just run the compare for the user.

Reply:
Yeah, I definitely like this better. Making the user mouse all the way over to the compare button and click it isn't a great user experience.

---
In reply to: [288312921](https://github.com/microsoft/azuredatastudio/pull/5690#discussion_r288312921) [](ancestors = 288312921)

Reply:
Added Compare button to the notification 
![image](https://user-images.githubusercontent.com/31145923/58515928-3ef8a380-815b-11e9-9ff0-e135a1e2c92e.png)